components/layout/app-bar.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { AppBar } from '@oacore/design'
+
+import { Consumer } from './context'
+import styles from './styles.css'
+
+import { AppBarToggle, Logo } from 'design'
+
+const DashboardAppBar = ({ children, className, ...restProps }) => (
+  <Consumer>
+    {({ sidebarId, sidebarOpen, toggleSidebar }) => (
+      <AppBar className={styles['app-bar']} fixed {...restProps}>
+        <AppBarToggle
+          className={styles['app-bar-toggle']}
+          type="button"
+          aria-haspopup="true"
+          aria-controls={sidebarId}
+          aria-expanded={sidebarOpen}
+          onClick={toggleSidebar}
+        />
+
+        <AppBar.Brand className={styles['app-bar-brand']} href="/">
```

Comment:
Everywhere else you use styles.element. It would be nice to stick with it. 
```suggestion
        <AppBar.Brand className={styles.appBarBrand} href="/">
```

Reply:
How do you do suggested change? 🤔

Reply:
It's pretty easy. Unfortunately, it works only for one line of code and it doesn't work in commits. You need to have open [all files](https://github.com/oacore/dashboard/pull/28/files).

https://twitter.com/github/status/1053360435074949120

Reply:
This is cool anyway! Thank you!